### PR TITLE
Kraken: Manage current_datetime in jormungandr for all api

### PR DESCRIPTION
**Pull request by azime**
Jira: http://jira.canaltp.fr/browse/NAVITIAII-1743

### CODE REVIEW

File: source/disruption/tests/disruption_test.cpp [hunk 1]
```diff
@@ -124,7 +124,7 @@ class Params {
 
 BOOST_FIXTURE_TEST_CASE(network_filter1, Params) {
     std::vector<std::string> forbidden_uris;
-    auto dt = "20131219T155000"_pts;
```
Comment (antoine-de) [inline source/disruption/tests/disruption_test.cpp]: why did you change this ?

File: source/jormungandr/tests/chaos_disruptions_tests.py [hunk 2]
```diff
@@ -105,6 +105,26 @@ def test_disruption_on_stop_area_b(self):
         eq_(channel['types'][0], 'web')
         eq_(channel['types'][1], 'email')
 
+    def test_current_datetime_out_of_bounds(self):
```
Comment (antoine-de) [inline source/jormungandr/tests/chaos_disruptions_tests.py:108]: :+1:

**Comment by azime** [comment]
corrections

**Comment by azime** [comment]
Jenkins, test this please
